lib/internal/backend/file/file.go

```diff
@@ -260,7 +276,7 @@ func getImageID(file *os.File, dockerURL *common.ParsedDockerURL, name string, d
 
 			if dockerURL == nil {
 				dockerURL = &common.ParsedDockerURL{
-					OriginalName: "",
+					OriginalName: appName + ":" + tag,
```

Comment:
I think that is equivalent to export an image by its raw sha, in which case you don't get a repositories file with name and tag.